Userland/Libraries/LibWeb/Layout/FlexFormattingContext.cpp

```diff
@@ -1213,14 +1218,24 @@ void FlexFormattingContext::distribute_any_remaining_free_space()
                 + space_between_items;
 
             if (is_direction_reverse()) {
-                item.main_offset = main_offset - item.main_size - item.margins.main_after - item.borders.main_after - item.padding.main_after;
-                main_offset -= amount_of_main_size_used;
+                dbgln("Direction is reverse");
```

Comment:
Do we need these `dbgln`s here? If so, can you hide them behind a `dbgln_if` macro?

Reply:
done

Reply:
Thanks for fixing; the rest of the changes should be reviewed by someone else :-)